src/applications/hca/components/FormFooter.jsx

```diff
@@ -15,11 +17,23 @@ export default class FormFooter extends React.Component {
       <div className="row">
         <div className="usa-width-two-thirds medium-8 columns">
           <div className="help-footer-box">
-            <h2 className="help-heading">Need help?</h2>
-            <GetFormHelp />
+            {!isHidden && (
```

Comment:
This is kind of a hack. I would have added to line 12 above, returning null if `isHidden` was true. But when this render function returns nothing we end up with not enough space between the bottom of the page content and the top of the footer navigation. But if we return all of this wrapper material, in particular the `.help-footer-box` div, we get the padding we need.